autovizwidget/autovizwidget/widget/utils.py

```diff
@@ -13,7 +13,7 @@ def infer_vegalite_type(data):
     data: Numpy array or Pandas Series
     """
 
-    typ = pd.lib.infer_dtype(data)
+    typ = pd.api.types.infer_dtype(data)
```

Comment:
Do we need to bump the minor required version of pandas? I don't know what pandas version this api got introduced with.

Reply:
Good catch. Looking at the pandas repository, it seems this change was made in 0.21.0.

Actually I just noticed that autovizwidget doesn't have an explicit dependency on _any_ version of pandas. Presumably it is relying on a transitive dependency through hdijupyterutils. Given that autovizwidget explicitly imports pandas, it seems that a dependency should be added, but I am asking to make sure before I make the change :)

Reply:
😢 Yes, it should have an explicit dependency on it and not rely on the transitive dependency, since it's calling it directly. Probably missed adding it at some point back in time.

Reply:
Done!